tools/clang/test/DXC/ftime-trace-granularity.hlsl

```diff
@@ -0,0 +1,12 @@
+// RUN: %dxc -E main -T vs_6_0 %s -ftime-trace -ftime-trace-granularity=99999 | FileCheck %s
```

Comment:
Would it be possible to add a comment here explaining what the two test cases are?  I think that it is checking that stdout works and that file output works?  Is that the idea?

Reply:
Yes.  That is the idea.  I can add some comments to the test.